package.json

```diff
@@ -2,6 +2,7 @@
   "version": "1.0.3",
   "scripts": {
     "build": "./build.sh",
+    "postinstall": "npm run build && open alfred-emoji.alfredworkflow",
```

Comment:
I'm not sold on this. If someone new wants to contribute, why would they want to immediately build a release after cloning? I would think they'd rather do the work they planned and _then_ build.

Reply:
Fair argument. Would you mind if I changed this to be tied to `npm run deploy` instead?  Open to script name suggestions here.

Reply:
Sounds good to me.

Reply:
> Fair argument.

Actually, I'm going to make a counter argument. the real intent here was that `npm install` would install everything if someone wanted to run a manual install, because they didn't trust the published `*.alfredworkflow` binary.

Nonetheless, I'm cool with just moving this to another script name.

Reply:
I understand the desire. But I think if you're going through the trouble of your own build then you likely don't need it done automatically for you. I would like to keep the development environment friendly.